packages/ui-predicate-vue/src/ui-predicate.vue

```diff
@@ -85,12 +95,13 @@ export default {
       this.$emit('changed', this.ctrl);
     },
   },
-  mounted() {
+  created() {
     const vm = this;
-
+    const { ui, ...columns } = this.config;
```

Comment:
oohhh d'accord ! Je comprends ta remarque de tout à l'heure, effectivement ici il y a un problème (parce que ce n'est pas explicite) est-ce qu'on ne peut pas partir là dessus plutôt : ?

```js
config:{
  targets:[],
  // [...]
  options:{
     ui:{}
  }
}
```

Reply:
Je suis d'avis de garder le même nom que celui utilisé dans le Core.

`data: this.data`, `columns: this.columns`, `options: this.options`;


`<ui-predicate
    :data="predicateModel"
    :columns="$options.UI_PREDICATE_COLUMNS"
    :options="$options.UI_PREDICATE_OPTIONS"
    @changed="onChange"/>`

Qu'en penses-tu ?